### ISSUE TITLE
Zenoss Core 5 Support

### ISSUE THREAD

**Issue by Rockstar04** [ZCA/zenoss-chef-cookbook#24]
It seems this cookbook has some open issues that deal with Core 4 support. Looking at [server.rb#L43-L52](https://github.com/ZCA/zenoss-chef-cookbook/blob/master/recipes/server.rb#L43-L52) it looks like this cookbook does not support Core 5 at all.

Are their plans to modernize the cookbook?

**Comment by BenLiyanage** [comment]
Yea, was just  wondering the same thing.  This cookbook seems pretty dead.

**Comment by dpetzel** [comment]
I don't believe that are any immediate plans. I personally am no longer working on it as I don't use Zenoss anymore

**Comment by Rockstar04** [comment]
Same here, We have moved on to other monitoring tools

**Comment by ewolkowicz** [comment]
We have also moved on to other monitoring platforms.  I started writing new providers to replace this cookbook but since we shifted to new monitoring platforms that went dead.  It 'mostly' works...
